ci/tests/puppeteer/run.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
+scenario="$1"
+if [[ -z "$scenario" ]] ; then
+  echo Usage: ./ci/tests/puppeteer/run.sh ${PWD}/ci/tests/puppeteer/scenarios/[scenario folder]
+  exit 1
+fi
+
+# note if debugging you might need to call
+# await page.setDefaultNavigationTimeout(0);
+DEBUG=${2:-debug}
```

Comment:
I am not sure we need this level of flexibility in such tests. If you need to debug something, you could modify the jvmArgs settings in the descriptor as a one-time thing only to activate debug, and then remove it later. Introducing 3 separate settings is not necessary.

Reply:
To clarify, I am certainly not against this change; I just don't think we would get much use out of it. I feel like it would be a much faster process if I run a CAS server separately in debug mode and just have a puppeteer script point to it, rather than rebuilding/restarting the whole puppeteer stack every time while it's being debugged.

Reply:
Setting up a test server is kind of a pain if aren't doing it all the time so I am willing to run a script that takes a little longer if it means I don't need to run a bunch of manual steps. The script could probably be broken up so as to make it easier to only run the parts needed but it's not too bad as it is. I added a if statement that only installs the node modules if they aren't already there. The debug arguments don't really need to be arguments and they have defaults so they could be removed if it ever became too cumbersome deal with them. I now see how the `jvmArgs` would work but it wasn't immediately obvious to me since there weren't any examples of it being used that I could find. I can remove those later.

Reply:
It seems like you might be better off having a custom script that automates the debug process for you, because I don't know if I would get much mileage out of this, and it's no different or more manual than running a CAS server in debug model. Also, installing the node_modules conditionally seems a rather dangerous thing specially as puppeteer gets updated. You need to install puppeteer globally to have it skip the modules installation, or have a custom shell script and/or an alias that does things for you. The script is meant to be run by CI, and there is no much value in having debug statements in there or conditional installation modules.